scenarioo-server/build.gradle

```diff
@@ -64,14 +64,14 @@ task printPathConfigInformationForNodeJSAndGrunt << {
 task createVersionPropertiesFile << {
 	def aggregatedDataFormatVersion = project.scenariooAggregatedDataFormatVersion
 	def apiVersion = project.scenariooApiVersion
-    def releaseBranch = project.scenariooReleaseBranch
+    def documentationVersion = project.documentationVersion
```

Comment:
Align with other def statements.

Reply:
@adiherzog this was just a renaming - I do not have time during such small changes to cleanup code that was wrongly aligned - AND this is only because the other lines use tabs and this one doesnt - my intellij handles it properly, it looks good - and I do not like to reformat code that I didnt have to touch in PRs - if we want to cleanup formatting we should do that as a separate task and then for all files.